admin-ui/plugins/smtp-management/components/SmtpManagement/SmtpForm.js

```diff
@@ -88,13 +88,14 @@ function SmtpForm({ item, handleSubmit }) {
       dispatch(testSmtp(opts))
     }
     else {
-      toast.error("All fields is required");
+      toast.error("Mandatory fields are required");
```

Comment:
if possible add `translation` for this message.

Reply:
Done